core/src/test/java/org/apache/iceberg/avro/TestAvroDeleteWriters.java

```diff
@@ -74,7 +75,7 @@ public void createDeleteRecords() {
 
   @Test
   public void testEqualityDeleteWriter() throws IOException {
-    OutputFile out = new InMemoryOutputFile();
+    OutputFile out = new InMemoryFileIO().newOutputFile(UUID.randomUUID().toString());
```

Comment:
Same comment here about saving and reusing the `InMemoryFileIO`, especially as this class uses it multiple times.